geomesa-web/geomesa-web-data/README.md

```diff
@@ -209,10 +203,20 @@ curl --header 'Accept: text/plain' --get --data-urlencode 'q=select mySft.myAttr
 | --- | ---- |
 | accumulo-core-1.5.2.jar | 3748459 |
```

Comment:
We should double check this list after the changes I made yesterday. I'll try to do it if I have time today and post another comment if I get to it.